lightning/src/ln/channel.rs

```diff
@@ -386,6 +386,8 @@ pub(super) struct Channel<ChanSigner: ChannelKeys> {
 	commitment_secrets: CounterpartyCommitmentSecrets,
 
 	network_sync: UpdateStatus,
+
+	supports_shutdown_anysegwit: bool,
```

Comment:
I don't think we want to cache this in the `Channel` - see my comment at https://github.com/lightningnetwork/lightning-rfc/pull/672#pullrequestreview-588879027 but I think we should be passing the `InitFeatures` object through the `shutdown` message handler (this may require a change to ChannelManager, PeerHandler, and the `ln::msgs::ChannelMessageHandler` trait, but it should be as easy as adding a new parameter.